assets/js/atomic/components/product-list/summary/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+const ProductListSummary = ( { className, product } ) => {
+	if ( ! product.description ) {
+		return null;
+	}
+
+	return (
+		<div
+			className={ classnames(
+				className,
+				'wc-block-grid__product-summary'
+			) }
+			dangerouslySetInnerHTML={ {
+				__html: product.description,
```

Comment:
I know we've talked about it before, but I still have this niggling little thing in the back of my head that we should look at client side escaping of this value.  This will especially become important if/when we start making things more extensible and/or we eventually expose these components for use in other ui/ux implementations (i.e. extension developers).  On its own, the component is heavily dependent on the assumption that `description` is already sanitized.

Reply:
Related to this, we are doing the same in the _Featured Product_ and _Featured Category_ blocks:

https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/blocks/featured-product/block.js#L316

So if we change this, we might want to change it in all places.

Reply:
This should really be tackled as a follow up since it affects more than just this block/PR.